src/cookbook/use-multi-signature-transactions.md

```diff
@@ -0,0 +1,261 @@
+---
+title: 'Use Multi-Signature Transactions | Cookbook'
+head:
+  - - meta
+    - name: description
+      content: 'Learn how to use multi-signature transactions in Iroha.'
+  - - meta
+    - name: keywords
+      content: 'multi-signature transactions'
+---
+
+# How to Use Multi-Signature Transactions
+
+The following example shows how to set up a rule that all transactions made
+by a specific account should also be signed by an additional key.
+
+## Overview
+
+<!-- maybe some extra introduction info here as well? -->
+
+1. Create an account, set up a second signature rule for it, and register a
+   new asset.
+2. Submit the transaction signed by the main account key, minting the
+   asset.
+3. Check that the transaction is not committed by observing the asset.
+4. Submit the same transaction signed by the second additional key.
+5. Check that the transaction is committed, and the asset is minted.
+
+## Example
+
+1. Let's create an account that would need to have an additional signature
+   for its transactions. Let's also create an asset that this account will
+   later mint in order to check that the rule about second signature works
+   as expected.
+
+   The code below covers the following steps:
+
+   - Create account `mad_hatter` in `wonderland` domain with `key_pair_1`.
+   - Assign `SignatureCheckCondition` to the created account. This will
+     enforce transactions from `mad_hatter@wonderland` to have an
+     additional signature by `key_pair_2`.
+   - Register a numeric asset `casomile#wonderland` of type `Quantity` and
+     `Infinite` mintability.
```

Comment:
The variable names vary depending on the language, e.g. `key_pair_1` vs. `keyPair1`, so I think we should make sure to not use them in text outside the code groups. Maybe if code needs clarifications, we could add comments in code.

```suggestion
   - Create account `mad_hatter` in `wonderland` domain using the first key pair.
   - Assign a *signature check condition* to the created account. This will
     enforce transactions from `mad_hatter@wonderland` to have an
     additional signature (*the second key pair*).
   - Register a numeric asset `casomile#wonderland` of type `Quantity` and
     `Infinite` mintability.
```